chunked_stream/lib/src/chunked_stream_iterator.dart

```diff
@@ -111,13 +114,8 @@ class _ChunkedStreamIterator<T> implements ChunkedStreamIterator<T> {
   /// If an error is encountered before reading [size] elements, the error
```

Comment:
```suggestion
  /// If an error is encountered before reading [size] elements, an error
```

Reply:
it is actually "the error" that was encountered that will be thrown :D